### PR TITLE
expandable fields: add system record

### PR DESCRIPTION
**Pull request by anikachurilova**
* closes https://github.com/zenodo/zenodo-rdm/issues/284

Replace non-existing "Expire" icon:
Screenshot 2023-05-24 at 18 01 05
Screenshot 2023-05-24 at 18 01 22

Capitalize the action name in request action modal:
Screenshot 2023-05-23 at 15 56 51

Request details page placeholder preview:
placeholdr req deta
Screenshot 2023-05-25 at 17 06 23

### CODE REVIEW

File: invenio_requests/assets/semantic-ui/js/invenio_requests/request/actions/RequestActionModal.js [hunk 1]
```diff
@@ -40,14 +40,21 @@ export class RequestActionModal extends Component {
 
     const currentModalOpen = modalOpen[modalId];
 
+    const capitalizeFirstLetter = (value) => {
```
Comment (kpsherva) [inline invenio_requests/assets/semantic-ui/js/invenio_requests/request/actions/RequestActionModal.js:43]: there is a css class doing this, use the class instead ;)

**Review by kpsherva — APPROVED** [review]

File: invenio_requests/assets/semantic-ui/js/invenio_requests/request/actions/RequestActionModal.js (hunk 1 above)
Comment (anikachurilova) [reply invenio_requests/assets/semantic-ui/js/invenio_requests/request/actions/RequestActionModal.js:43]: Do you mean `text-transform: capitalize;`. It capitalizes all the words in the string, which will make it "Accept Request"

**Review by alejandromumo — COMMENTED** [review]
LGTM, however I'll let @kpsherva  answer your question about the .css class before accepting

File: invenio_requests/assets/semantic-ui/js/invenio_requests/request/actions/RequestActionModal.js (hunk 1 above)
Comment (kpsherva) [reply invenio_requests/assets/semantic-ui/js/invenio_requests/request/actions/RequestActionModal.js:43]: you could fix it by creating and using this class:
```
.capitalize-first-char{
display: inline-block;
 &::first-letter{
   text-transform: capitalize;
 }
}
```

Comment (anikachurilova) [reply invenio_requests/assets/semantic-ui/js/invenio_requests/request/actions/RequestActionModal.js]: Fixed https://github.com/inveniosoftware/invenio-app-rdm/pull/2225/files#diff-c4c330faab677d7c41b090ce5d9526ed283ca48128cfc1dc66c2d2b3f9522ab0R466

**Review by rekt-hard — APPROVED** [review]

**Review by kpsherva — APPROVED** [review]

**Pull request merged**